src/components/MessageList/MessageList.jsx

```diff
@@ -126,8 +126,9 @@ const mapStateToProps = state => ({
   messages: state.chat.messages.sort((curr, next) => (
     new Date(curr.timestamp) - new Date(next.timestamp)
   )),
+  typing: state.chat.typing,
```

Comment:
I dont think we need to pull this in, is this new?

Reply:
yes,  It's used:

`<li style={{ display: chat.typing ? 'block' : 'none' }} className="Message__client typing">`

Reply:
Thats chat.typing, not the state.chat.typing